### DISCUSSION TITLE
SUPER key

### DISCUSSION THREAD

**Discussion by angrycsm** [LingmoOS/Community#28] [Bugs Report/缺陷提交] [answered]
Sorry, I don't know how to speak English, I'm using a translator, I recently installed the version and my SUPER or WINDOWS key to open the applications screen doesn't work, how can I solve this? It's some keybind error, I tested other Linux versions, and they work with the SUPER key to show the applications tab, I didn't want to have to keep clicking with the mouse to open the applications.

**Comment by samwhelp** [comment, marked as answer]
## Config File

> There are two config files about keybind

| Config File |
| ----------- |
| [~/.config/kglobalshortcutsrc](https://github.com/samwhelp/lingmo-adjustment/blob/main/prototype/main/lingmo-config/locale/en_us/Sample-Lingmo-Light/asset/overlay/etc/skel/.config/kglobalshortcutsrc) |
| [~/.config/lingmoglobalshortcutsrc](https://github.com/samwhelp/lingmo-adjustment/blob/main/prototype/main/lingmo-config/locale/en_us/Sample-Lingmo-Light/asset/overlay/etc/skel/.config/lingmoglobalshortcutsrc#L17-L19) |

> There is one config file about mousebind

| Config File |
| ----------- |
| [~/.config/kwinrc](https://github.com/samwhelp/lingmo-adjustment/blob/main/prototype/main/lingmo-config/locale/en_us/Lingmo-Light/asset/overlay/etc/skel/.config/kwinrc#L50-L52) |

> you can check source code for mousebind action

| Source Code |
| ----------- |
| [GitHub / KDE / kwin / src / kwin.kcfg](https://github.com/KDE/kwin/blob/master/src/kwin.kcfg#L8-L60) |
| [invent.kde.org / kwin / src / kwin.kcfg](https://invent.kde.org/plasma/kwin/-/blob/master/src/kwin.kcfg?ref_type=heads#L8-L60) |

> for example: `~/.config/kwinrc`

``` ini
[MouseBindings]
CommandAll1=Move
CommandAll2=Toggle raise and lower
CommandAll3=Resize
CommandAllKey=Meta
CommandAllWheel=Change Opacity
CommandTitlebarWheel=Shade/Unshade
```




## About Keybind for Super Key

> [~/.config/lingmoglobalshortcutsrc](https://github.com/samwhelp/lingmo-adjustment/blob/main/prototype/main/lingmo-config/locale/en_us/Sample-Lingmo-Light/asset/overlay/etc/skel/.config/lingmoglobalshortcutsrc#L17-L19)

``` ini
[Meta]
Comment=Launcher
Exec=lingmo-launcher
```

> `Meta` means `Super` or `Win`

> But it doesn't work on my system.

> You can change it as follows




## Keybind Sample 1

> Keybind: `Super+p` would launch `lingmo-launcher`

``` ini
[Meta%2BP]
Comment=Launcher
Exec=lingmo-launcher
```




## Keybind Sample 2

> Keybind: `Alt+F1` would launch `lingmo-launcher`

``` ini
[Alt%2BF1]
Comment=Launcher
Exec=lingmo-launcher
```

**Comment by elysia-best** [comment]
Thank you for your report. Currently we don't have any key binding to launch the App launcher quickly. 

We could consider adding it in the next version. You can try to use a third-party software to work for you. 

Thx.
